Documentation/README.md

```diff
@@ -8,7 +8,14 @@ Generally dates are stored as a Unix Epoch times in milliseconds.
 Responses are served in JSON format.
 
 ## UUID Parameters
-All uuid parameters are expected to have the UUID without dashes.
+UUID's will work the same with or without dashes, but it is recommended to format it without dashes.
+
+``Recommended Format: f7c77d999f154a66a87dc4a51ef30d19``
+
+``Unrecommended Format: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``
+
+## Where Can I Find My UUID?
+The most efficient way is to go on [NameMC](https://namemc.com/), search up the desired profile, and it will show up under 'Minecraft Profile'. [Click here for a visual](https://prntscr.com/vw3f7d).

```

Comment:
Do you think this section should be added?

Reply:
Not sure, but anyway wording it like `Where can I find someone's UUID?` is probably better, also you should remove the ad from screenshot.

Reply:
Good point lol